modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -640,20 +640,19 @@ func (suite *KeeperTestSuite) TestWriteChannelUpgradeAck() {
 			path = ibctesting.NewPath(suite.chainA, suite.chainB)
 			suite.coordinator.Setup(path)
 
-			// create upgrade proposal with different version in init upgrade to see if the WriteUpgradeAck overwrites the version
```

Comment:
this looked odd to me when I was looking at it. `WriteUpgradeAck` will always overwrite w whatever version we supplied. Lmk if I'm not grabbing some edge case here that should be covered.